packages/botonic-cli/src/util/bot-config.ts

```diff
@@ -46,6 +63,49 @@ export class BotConfig {
         botonic_cli_version: botonicCliVersion,
       },
       packages,
+      tools: configLoaded.tools,
+      payloads: configLoaded.payloads,
+      webviews: configLoaded.webviews,
+    }
+  }
+  static async getTools(appDirectory: string) {
+    try {
+      const botConfigPath = path.join(appDirectory, '.src/bot-config.ts')
```

Comment:
we must use path utils to create this path: `'.src/bot-config.ts'`, otherwise could not work in all operative systems. Use constants for bot config file. Was `bot-config.ts` or `bot.config.ts`?

Reply:
I removed this function because I wasn't using it at all